cpg-core/src/main/java/de/fraunhofer/aisec/cpg/frontends/Language.kt

```diff
@@ -50,6 +50,12 @@ abstract class Language<T : LanguageFrontend> : Node() {
     /** The namespace delimiter used by the language. Often, this is "." */
     abstract val namespaceDelimiter: String
 
+    /**
+     * Some languages can have other strings where we may want to split a name (e.g., "." in
+     * addition to "::")
+     */
+    open val nameSplitter: Array<String> = emptyArray()
```

Comment:
tbh I'm not sure if it's *really* necessary. I think I added it to give us more freedom (just in case we need it one day in the future) without hardcoding a `".`" as generic split character/string.

Reply:
This is the last comment. I am still not quite convinced about this feature, we are also only storing one delimiter in the `Name` class itself, so I guess this will add to more confusion. Is it ok if I remove it?

Reply:
Feel free to remove it